datumaro/plugins/open_images_format.py

```diff
@@ -238,6 +289,53 @@ def _load_labels(self, items_by_id):
                     item.annotations.append(Label(
                         label=label_index, attributes={'score': confidence}))
 
+    def _load_bboxes(self, items_by_id):
+        label_categories = self._categories[AnnotationType.label]
+
+        for bbox_path in self._glob_annotations(
+            '*' + OpenImagesPath.BBOX_DESCRIPTION_FILE_SUFFIX
+        ):
+            with self._open_csv_annotation(bbox_path) as bbox_reader:
+                for bbox_description in bbox_reader:
+                    image_id = bbox_description['ImageID']
+                    item = items_by_id[image_id]
+
+                    label_name = bbox_description['LabelName']
+                    label_index, _ = label_categories.find(label_name)
+                    if label_index is None:
+                        raise UndefinedLabel(
+                            item_id=item.id, subset=item.subset,
+                            label_name=label_name, severity=Severity.error)
+
+                    if item.has_image:
+                        height, width = item.image.size
```

Comment:
Check if size is available with `item.image.has_size`.

Reply:
`has_size` won't help here, because it doesn't attempt to load the size from the image file if it isn't already loaded. Added a check for `image.size is not None` instead.